doc/api/prev_api_changes/api_changes_1.5.0.rst

```diff
@@ -34,48 +34,48 @@ before that gets overhauled).
 Tightened input validation on 'pivot' kwarg to quiver
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Tightened validation so that only {'tip', 'tail', 'mid', and 'middle'}
-(but any capitalization) are valid values for the 'pivot' kwarg in
-the `Quiver.__init__` (and hence `Axes.quiver` and
-`plt.quiver` which both fully delegate to `Quiver`).  Previously any
-input matching 'mid.*' would be interpreted as 'middle', 'tip.*' as
-'tip' and any string not matching one of those patterns as 'tail'.
+Tightened validation so that only {'tip', 'tail', 'mid', and 'middle'} (but any
+capitalization) are valid values for the *pivot* keyword argument in the
+`.Quiver` class (and hence `.axes.Axes.quiver` and `.pyplot.quiver` which both
+fully delegate to `.Quiver`).  Previously any input matching 'mid.*' would be
```

Comment:
needs quoting to avoid asterisk being interpreted as italic (just below too)

Reply:
That's only true on GitHub; it's correct in [the rendered result](https://24353-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/prev_api_changes/api_changes_1.5.0.html#tightened-input-validation-on-pivot-kwarg-to-quiver).